### DISCUSSION TITLE
Strange bug visible when using Neovide

### DISCUSSION THREAD

**Discussion by Readf0x** [LazyVim/LazyVim#4285] [Q&A] [answered]
https://github.com/user-attachments/assets/49348a7a-f697-4dba-b069-746e10d544ed

When I open the command line the cursor starts jumping back and forth. Anyone know why this happens? And where should I report this (if at all)?

**Comment by dpetka2001** [comment, marked as answer]
This might be most probably due to `noice.nvim` i believe. I've seen an open issue at the plugin's repo, but since the maintainer is away until end of August, you'll have to wait for a fix until after he comes back. I've not experienced this in Neovim itself, but others reported same issue in Neovide.
